### ISSUE TITLE
Make Polling Rate Configuraable

### ISSUE THREAD

**Issue by danthony06** [swri-robotics/gps_umd#55]
```
timer_ = create_wall_timer(1s, std::bind(&GPSDClientComponent::step, this));
```

Creates a 1 second polling timer. This polling period should be configurable.

**Comment by packwidth** [comment]
@danthony06, the PR above makes the polling rate configurable through parameters which can be passed at node construction directly or through a config file, or through `ros2 param`

**Comment by danthony06** [comment]
Yes, I left a review comment about the changes that needs to be addressed.

**Comment by danthony06** [comment]
Sorry, the review didn't get published. I just made my comments available.
